Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1503,111 +1043,49 @@ - (void)initUserSessionAndCallCallback:(BOOL)callCallback {
 }
 
 - (void)initializeSession {
-    if (![self hasBranchKey] && ![self hasAppKey]) {
+    if (!self.branchKey) {
```

Comment:
App Key is still supported, look at the getInstance and getTestInstance methods -- both do fallback on the getAppKey method. The `self.branchKey` property is only filled in once, but can have either a Branch Key or an App Key.